iis-workflows/iis-workflows-ingest-pmc/src/test/java/eu/dnetlib/iis/workflows/ingest/pmc/metadata/PmcXmlHandlerTest.java

```diff
@@ -58,35 +60,91 @@ public void testParsing() throws Exception {
 			reader.setFeature("http://xml.org/sax/features/validation", false);
 			reader.setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false);
 			reader.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
-			
+
 			ExtractedDocumentMetadata.Builder metaBuilder = ExtractedDocumentMetadata.newBuilder();
 			metaBuilder.setId("some-id");
 			metaBuilder.setText("");
 			PmcXmlHandler pmcXmlHandler = new PmcXmlHandler(metaBuilder);
-			saxParser.parse(inputStream = PmcXmlHandler.class.getResourceAsStream(filePath), 
-					pmcXmlHandler);
-			ExtractedDocumentMetadata meta = metaBuilder.build();
-			assertEquals("research-article", meta.getEntityType());
-			assertEquals("BMC Systems Biology", meta.getJournal());
-			assertEquals("111", meta.getPages().getStart());
-			assertEquals("111", meta.getPages().getEnd());
-			
-			assertNotNull(meta.getReferences());
-			assertEquals(34, meta.getReferences().size());
-			
-			assertNotNull(meta.getAffiliations());
-			assertEquals(2, meta.getAffiliations().size());
-			
+			saxParser.parse(inputStream = PmcXmlHandler.class.getResourceAsStream(filePath), pmcXmlHandler);
+			checkJats10Metadata(metaBuilder.build());
+		} finally {
+			if (inputStream != null) {
+				inputStream.close();
+			}
```

Comment:
If inputStream would be declared as class variable, then it could be closed in `@After` section of junit test.
Then we could omit try/finally block in test methods